MAUI/NumericEntry/Overview.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: About .NET MAUI NumericEntry control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI NumericEntry (SfNumericEntry) control, its features, and more.
+platform: MAUI
+control: SfNumericEntry
+documentation: ug
+---
+
+# .NET MAUI NumericEntry Overview
+
+The .NET MAUI NumericEntry control is designed to provide a user-friendly and advanced input experience for numeric values. It supports a wide range of numeric formats, including currency, percentage, and decimals, among others. With its key features, it enables enhanced user experience and facilitates input validation.
+
+## Control Structure
+
+![.NET MAUI NumericEntry structure](Overview_images/overview_img.png)
+
+## Key Features
+
+* Input validation triggered when focus is lost or the enter key is pressed.
```

Comment:
* Input validation is triggered when the focus is lost, or the enter key is pressed.

Reply:
The requested content modification has been successfully addressed